lib/components/ConnectModal/index.tsx

```diff
@@ -1,6 +1,6 @@
 import { cx } from "@emotion/css";
+import AccountBalanceWalletIconDefault from "@mui/icons-material/AccountBalanceWallet";
```

Comment:
```suggestion
import AccountBalanceWalletIcon from '@mui/icons-material/AccountBalanceWallet';
```

Should do the trick, no?

Reply:
@sprrwhwk There is two other icon imports in this file `Close` and `MailOutline`, the reason I added the "Default" text to the end of the string was to be consistent with the naming of the other two imports, and below due to esm import issues, default needs to be called on the icons after importing.

```
import MailOutlineIconDefault from "@mui/icons-material/MailOutline";
import AccountBalanceWalletIconDefault from "@mui/icons-material/AccountBalanceWallet";

// ems modules and javascript are strange so we import like this
const MailOutlineIcon = (MailOutlineIconDefault as any).default as any;
const AccountBalanceWalletIcon = (AccountBalanceWalletIconDefault as any)
  .default as any;
```